### PR TITLE
feat(zkstack): Separate chain deploy l2 contracts

### PR DESCRIPTION
**Pull request by matias-gonz**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zkstack dev fmt` and `zkstack dev lint`.

### CODE REVIEW

File: zkstack_cli/crates/config/src/chain.rs [hunk 1]
```diff
@@ -172,6 +174,7 @@ impl ChainConfig {
             wallet_creation: self.wallet_creation,
             legacy_bridge: self.legacy_bridge,
             evm_emulator: self.evm_emulator,
+            l1_wallets_path: Some(self.l1_wallets_path.clone()),
```
Comment (Deniallugo) [inline zkstack_cli/crates/config/src/chain.rs:177]: why is it separate?

File: zkstack_cli/crates/zkstack/src/commands/chain/deploy_l2_contracts.rs [hunk 2]
```diff
@@ -45,88 +42,45 @@ pub async fn run(
...
-    let era_chain_id = ecosystem_config.era_chain_id;
-    let wallets = ecosystem_config.get_wallets()?;
+    let mut contracts = chain.get_contracts_config()?;
+    let era_chain_id = ecosystem
+        .as_ref()
+        .map_or(get_default_era_chain_id(), |e| e.era_chain_id);
+    let wallets = ecosystem.as_ref().map_or(
+        WalletsConfig::read(shell, chain.l1_wallets_path.clone()),
+        |e| e.get_wallets(),
+    )?;
```
Comment (Deniallugo) [inline zkstack_cli/crates/zkstack/src/commands/chain/deploy_l2_contracts.rs:55]: if we need wallets, from ecosystem, probably this function doesn't belong here.

Comment (Deniallugo) [reply zkstack_cli/crates/zkstack/src/commands/chain/deploy_l2_contracts.rs:55]: @sanekmelnikov What have you done in your pr?

**Comment by matias-gonz** [comment]
Should move deploy l2-contracts to ecosystem

**Pull request closed without merging**